test/multiverse/suites/active_support_logger/Envfile

```diff
@@ -5,6 +5,9 @@
 instrumentation_methods :chain, :prepend
 
 ACTIVE_SUPPORT_VERSIONS = [
+  ['8.0.0', 3.2],
+  ['7.2.2', 3.1],
+  ['7.1.5', 2.7],
```

Comment:
ActiveSupport::Logger.broadcast is not defined on Rails versions above 7.0.
The tests fail because we call `ActiveSupport::Logger.broadcast` in the setup, regardless of the version running the test. 

We can either test on newer versions and update the setup to gracefully fail when we're using a version where it isn't supported, or just skip testing on Rails versions above 7.0. 

```suggestion
# ActiveSupport::Logger.broadcast is only defined on Rails 7.0 and below
# Do not test newer Rails versions on this suite.
ACTIVE_SUPPORT_VERSIONS = [
```

Reply:
@tannalynn or @hannahramadan, do you have a preference on the approach?

Reply:
Is there any reason why we'd need to run these tests on versions that don't do the broadcast? Because if there isn't a specific reason, it makes more sense to just not run them on those versions

Reply:
oh also, Do we install the instrumentation only on the versions that support broadcast?